backend/exchanges/btcdirect.go

```diff
@@ -76,3 +89,21 @@ func BtcDirectDeals() *ExchangeDealsList {
 		},
 	}
 }
+
+// BtcDirectInfo returns the information needed to interact with BtcDirect.
+func BtcDirectInfo(action ExchangeAction, acct accounts.Interface) btcDirectInfo {
+	res := btcDirectInfo{
+		Url:    btcUrl,
+		ApiKey: btcDirectProdAPiKey,
+	}
+
+	if _, isTestnet := coin.TestnetCoins[acct.Coin().Code()]; isTestnet {
+		res.ApiKey = btcDirectTestApiKey
+	}
+
+	if action == BuyAction {
+		addr := acct.GetUnusedReceiveAddresses()[0].Addresses[0].EncodeForHumans()
```

Comment:
I don't like it much to get the first script type in the addresses list without checking which one it is, but I see that we do the same for Moonpay, so I guess it is okay.. wdyt @benma ?

Reply:
Same here as before, just copied from Moonpay, open to suggestions!

Reply:
Yeah, if we change it we should change it for both in a different PR.